crates/sargon/src/wrapped_radix_engine_toolkit/high_level/sargon_specific_types/transaction/transaction_status.rs

```diff
@@ -0,0 +1,60 @@
+use crate::prelude::*;
+
+#[derive(
+    Clone,
+    Debug,
+    Serialize,
+    Deserialize,
+    PartialEq,
+    Eq,
+    Hash,
+    Ord,
+    PartialOrd,
```

Comment:
Do we really need the ordering?

Reply:
definitely not, removing as per you comment on meeting yesterday.